### PR TITLE
Shop Styles

### PR DESCRIPTION
**Pull request by jsburgin**

### CODE REVIEW

File: app/src/main/java/net/jmhossler/roastd/shoptask/ShopActivity.java [hunk 1]
```diff
@@ -83,15 +94,20 @@ public void displayAddress(String address) {
   }
 
   @Override
-  public void displayMapsUrl(String url) {
-    mMapsUrl.setText(url);
+  public void createMapsLink(String url) {
+    mAddressBox.setOnClickListener(v -> startActivity(new Intent(Intent.ACTION_VIEW, Uri.parse(url))));
```
Comment (jmhossler) [inline app/src/main/java/net/jmhossler/roastd/shoptask/ShopActivity.java]: I'm currently working on fixing the bug that happens when you click this. We should consider that outside the scope of this PR.

**Comment by jsburgin** [comment]
Going to include the drink and bean styles since they're very similar.

**Comment by jsburgin** [comment]
And bean:

**Pull request merged**